block/block.go

```diff
@@ -48,6 +48,18 @@ type Blockchain struct {
 	tail map[sig.Hash]Commit
 }
 
+func NewBlockchain() Blockchain {
+	genesis := Genesis()
+	return Blockchain{
+		head: Commit{
+			Polka: Polka{
+				Block: &genesis,
+			},
+		},
+		tail: map[sig.Hash]Commit{},
```

Comment:
Genesis block also needs to be included in the `tail` since the `head` must be included in here at all times. This means we should probably rename `tail` to `blocks` as well since it's not really a tail if it includes the head.

Reply:
Fixed in https://github.com/renproject/hyperdrive/pull/5